controllers/chat/ChatController.js

```diff
@@ -340,7 +340,9 @@ module.exports = {
 
       const response = {
         ...chat,
-        members: targetsUserModel
+        members: targetsUserModel,
+        betterChannelMember,
+        betterChannelMemberObject
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `betterChannelMember` and `betterChannelMemberObject` to the response object seems to be a logical change, given that these variables are now being returned from the `updateBetterChannelMembers` function. However, it would be beneficial to ensure that the client-side code can handle these new properties in the response object.
```diff
        const response = {
          ...chat,
         members: targetsUserModel,
+        betterChannelMember,
+        betterChannelMemberObject
        };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->